### ISSUE TITLE
Enum names are converted to uppercase

### ISSUE THREAD

**Issue by szagi3891** [poem-web/poem#122]
## Expected Behavior
I would expect this to be case-sensitive, as defined in the rust code

## Actual Behavior
Enum names are converted to uppercase

## Steps to Reproduce the Problem

1. You have to run this test code:

```
#[derive(Clone, Debug, PartialEq, Enum)]
enum SelectionDisplayType {
    Column,
    Row,
    TwoColumns,
    ThreeColumns,
    CorrectScore
}

#[derive(Clone, Debug, PartialEq, Object)]
struct RabViewDetailsModel {
    displayOrder: u64,
    selectionDisplayType: SelectionDisplayType,
    active: bool,
}

#[test]
fn decode_enum() {
    let data = r#"{"displayOrder":2,"active":true,"selectionDisplayType":"CorrectScore"}"#;

    let aa = serde_json::from_str::<RabViewDetailsModel>(&data);
    println!("ddd {aa:#?}");

    let g = RabViewDetailsModel {
        displayOrder: 33,
        selectionDisplayType: SelectionDisplayType::CorrectScore,
        active: true
    };

    let hh = serde_json::to_string(&g);
    println!("hh {hh:#?}");
}
```

This test produces a result on the console:
```
ddd Err(
    Error("Expected input type \"SelectionDisplayType\", found \"CorrectScore\". (occurred while parsing \"RabViewDetailsModel\")", line: 0, column: 0),
)
hh Ok(
    "{\"active\":true,\"displayOrder\":33,\"selectionDisplayType\":\"CORRECT_SCORE\"}",
)
```

## Specifications

- Version: poem = "1.0.38", poem-openapi = "1.0.41"
- Platform: MacOs
- Subsystem:

**Comment by szagi3891** [comment]
I see that there is an attribute: 
#[oai(rename_all="PascalCase")]

Maybe it would be a good idea to make this option the default ?

**Comment by sunli829** [comment]
Fixed in `v1.0.46` 🙂

**Comment by szagi3891** [comment]
Thank you very much :)
